shuup/front/templates/shuup/front/product/detail_order_section.jinja

```diff
@@ -1,3 +1,5 @@
 {%- from "shuup/front/macros/product.jinja" import render_product_order_section with context -%}
 
+{% if not product.is_subscription_parent() %}
 {{ render_product_order_section(product, variation_variables, variation_children, unorderability_reason, orderable_variation_children) }}
+{% endif %}
```

Comment:
I don't think this `if` is needed, since `render_product_order_section` should work fine even though there is no ProductOrderForm which handles subscription products. It should then just render nothing.